### ISSUE TITLE
Why some windows don't have a colorful header bar?

### ISSUE THREAD

**Issue by Leatherpants** [snwh/paper-gtk-theme#272]
I realize that many applications have thier own css file in the apps folder that define thier header-bar color, but that doesn't work. For example, i saw gnome-documents.css, gnome-software.css and gnome-music.css but when i open those applications i saw thier header-bars are all grey.

**Comment by snwh** [comment]
It depends on which desktop environment/window manager you are using.

**Comment by Leatherpants** [comment]
thank you for taking time to read my issue. I use GNOME 3.18 and I think those files should work in GNOME. I want to fix it myself, but I don't know how.

**Comment by jwcone** [comment]
Note that the @import statements for gnome-music.css and gnome-software.css are both commented out in apps.css (at least in the version of Paper that I cloned a day or two ago).

However, the @import for gnome-documents.css is not commented out in apps.css, and it still shows up as grey when I open it.  Another example of an app with css that doesn't seem to be working as expected is cheese.  Other apps are coming up with the colored header bars: archive-manager, web/epiphany, gnome-calculator, gedit, gnome-disks, logs, et cetera.  Not sure what the difference is. (I'm on GNOME 3.18.2)

**Comment by snwh** [comment]
It may be the case that things may have changed for the GNOME apps and I may no longer be able to theme them, which is very likely. The method I was using was very hacky.

**Comment by seyDoggy** [comment]
I just installed from the ppa and used Gnome Tweak to set the GTK+, Icons and Cursor all to Paper. I do not get the multicolored Window decorations. I have GNOME Shell 3.16.4 and Linux Kernel 4.2.0-35-generic.

```sh
 % gnome-shell --version                                                 11:01
GNOME Shell 3.16.4

 % uname -a                                                              11:06
Linux 7450 4.2.0-35-generic #40-Ubuntu SMP Tue Mar 15 22:15:45 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
```

![screenshot from 2016-04-20 11-16-33](https://cloud.githubusercontent.com/assets/314795/14679879/ba54cc7e-06e9-11e6-80d4-3e95b3fe339d.png)

**Comment by snwh** [comment]
@seyDoggy correct. Not all apps will have colourful header bars, it's a select few.
